### ISSUE TITLE
Migrate to new Firefox permissions

### ISSUE THREAD

**Issue by nathan-b** [nathan-b/colortoggle#24]
Firefox extensions now can request permissions rather than having access to everything within the browser. Modify Color Toggle to request only the permissions it needs.

**Comment by Reaversword** [comment]
This is about we can't set up Color Toggle now in Firefox 52?. (Acept button doesn't works!).

Glad to see you're on that, Color Toggle is very important to not burn our eyes with nuclear-white backgrounds.

Thanks!.

**Comment by jmpedron** [comment]
It is not currently working with Firefox 52.0

**Comment by nathan-b** [comment]
I don't believe the permissions issue is related to why the extension is not working on FF 52, but I could be wrong because I have not had the chance to investigate. I hope I can find a way to get it working again.

I've filed bug #25 regarding this issue. If it turns out to be the same as the permissions problem described in this bug, I will dup.

**Comment by Reaversword** [comment]
I've in fact, returned to Firefox 51. Not without Color Toggle, mozilla.
